pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -131,3 +131,109 @@ type AccountList struct {
 func init() {
 	SchemeBuilder.Register(&Account{}, &AccountList{})
 }
+
+// Helper Functions
+
+//IsFailed returns true if an account is in a failed state
+func (a *Account) IsFailed() bool {
+	return a.Status.State == string(AccountFailed)
```

Comment:
This has technically changed, and continues to work only because the `AccountFailed` `AccountConditionType` in this package has the same string value as the `AccountFailed` const in `pkg/controller/account` defined in account_controller.go. Similar comment applies to the other methods below.

I've mentioned [before](https://github.com/openshift/aws-account-operator/pull/425#discussion_r459042894) that it's awkward/weird to have those strings duplicated for `Reason` and `Status`. I'm not sure if this is the right place to address that, though.